packages/commonwealth/client/scripts/views/pages/NotificationSettings/index.tsx

```diff
@@ -0,0 +1,179 @@
+import { CommunityAlert, ThreadSubscription } from '@hicommonwealth/schemas';
+import { useFlag } from 'hooks/useFlag';
+import useUserLoggedIn from 'hooks/useUserLoggedIn';
+import React, { useCallback, useState } from 'react';
+import app from 'state';
+import { useCommunityAlertsQuery } from 'state/api/trpc/subscription/useCommunityAlertsQuery';
+// eslint-disable-next-line max-len
+import { useRegisterClientRegistrationTokenMutation } from 'state/api/trpc/subscription/useRegisterClientRegistrationTokenMutation';
```

Comment:
we should create index.ts file in api/trpc/subscription that would import and export all the hooks, then, you could simply do 

`import { useRegisterClientRegistrationTokenMutation } from 'state/api/trpc/subscription/';`

Reply:
This kind of defeats the point because the idea was that we would wrap them with logic and I think throwing them all into the same file is probably not the best idea.